Packages/com.unity.inputsystem/InputSystem/Editor/Internal/InputActionSerializationHelpers.cs

```diff
@@ -185,6 +262,47 @@ private static string FindUniqueName(SerializedProperty arrayProperty, string ba
 
             return result;
         }
+
+        public static void RenameAction(SerializedProperty actionProperty, SerializedProperty actionMapProperty, string newName)
+        {
+            var nameProperty = actionProperty.FindPropertyRelative("m_Name");
+            var bindingsProperty = actionMapProperty.FindPropertyRelative("m_Bindings");
+            for (var i = 0; i < bindingsProperty.arraySize; i++)
+            {
+                var element = bindingsProperty.GetArrayElementAtIndex(i);
+                var actionNameProperty = element.FindPropertyRelative("action");
+                if (actionNameProperty.stringValue == nameProperty.stringValue)
+                {
+                    actionNameProperty.stringValue = newName;
+                }
+            }
+            nameProperty.stringValue = newName;
+        }
+
+        public static void RenameActionMap(SerializedProperty actionMapProperty, string newName)
+        {
+            var nameProperty = actionMapProperty.FindPropertyRelative("m_Name");
+            nameProperty.stringValue = newName;
+        }
+
+        public static void RenameComposite(SerializedProperty actionMapProperty, string newName)
```

Comment:
`actionMapProperty` probably copypaste here. What does this function do?

Reply:
It's meant for renaming the composite group. It's how designed the UI for now.